ui/main.qml

```diff
@@ -490,33 +501,45 @@ StatusWindow {
         }
     }
 
-    MacTrafficLights { // FIXME should be a direct part of StatusAppNavBar
+    // TODO: This area for MacOs and generic topbar
+    // for with notification and Profile button should probably
+    // be combined here
+    Item {
+        visible: d.showCustomMacOsToolbar
+
         anchors.left: parent.left
+        anchors.right: parent.right
         anchors.top: parent.top
-        anchors.margins: 13
-
-        visible: Qt.platform.os === Constants.mac && applicationWindow.visibility !== Window.FullScreen
 
-        onClose: {
-            if (loader.sourceComponent != app) {
-                Qt.quit()
-                return
-            }
-
-            if (localAccountSensitiveSettings.quitOnClose) {
-                Qt.quit();
-                return
-            }
+        implicitHeight: 40
```

Comment:
Can you get the height from the macOS api? you could expose it as a property from SystemUtils